### PR TITLE
Added p-values to spearman' rank and pearson's rho #340

### PR DESCRIPTION
**Pull request by madhavmk**
This adds the features requested in issue #340 .
I have modified the file chart.js only.
The p value is calculated from the method mentioned in the issue thread.

### CODE REVIEW

**Comment by madhavmk** [comment]
I have created a PR: #388.
If any changes have to be made, please tell me.

**Comment by jingchunzhu** [comment]
you can see the white space inconsistency in the diff, making the PR code formatting is off from existing code.   We set tab = 4 white space. I suspect yours is different.

**Comment by madhavmk** [comment]
@jingchunzhu I have updated the PR to have uniform tabs of size 4. I also removed some extra comments which were unnecessary.

**Comment by jingchunzhu** [comment]
@madhavmk please see issue #340 comments regarding validation of the p value using an independent method

**Comment by madhavmk** [comment]
@jingchunzhu I have calculated p values with Excel functions independently and they match the p values I'm getting in the xena-client. Please see the screenshots I have attached in issue #340.

File: js/chart.js [hunk 1]
```diff
@@ -1306,11 +1305,10 @@ function render(root, callback, sessionStorage) {
 						xlabel + ' ~ ' + ylabel + '<br>' +
 						'Pearson\'s rho<br>' +
 						'r = ' + rho.toPrecision(4) + ' , ' +
-                        'p = ' + pValueRho.toPrecision(4) + '<br>' +
-                        'Spearman\'s rank rho<br>' +
+						'p = ' + pValueRho.toPrecision(4) + '<br>' +
+						'Spearman\'s rank rho<br>' +
 						'ρ = ' + spearmanRho.toPrecision(4) + ' , ' +
```
Comment (jingchunzhu) [inline js/chart.js]: change "ρ" to "rho".  ρ and p look too hard to distinguish from each other, which will confuse users.

File: js/chart.js [hunk 2]
```diff
@@ -1306,11 +1305,10 @@ function render(root, callback, sessionStorage) {
 						xlabel + ' ~ ' + ylabel + '<br>' +
 						'Pearson\'s rho<br>' +
 						'r = ' + rho.toPrecision(4) + ' , ' +
-                        'p = ' + pValueRho.toPrecision(4) + '<br>' +
-                        'Spearman\'s rank rho<br>' +
+						'p = ' + pValueRho.toPrecision(4) + '<br>' +
```
Comment (jingchunzhu) [inline js/chart.js]: change format to the following example
 r= 0.38 (p = 0.00016)

**Comment by madhavmk** [comment]
@jingchunzhu I have updated the PR to reflect on the formatting changes you had requested.
If any changes have still have to be made, please inform me.

**Comment by madhavmk** [comment]
@jingchunzhu Is there anything wrong with the formatting changes I made? I tried to match it with what you had requested above. If anything else has to be modified, please inform me and I will change it.

**Pull request merged**